### PR TITLE
EN-7260: added output directory flag

### PR DESCRIPTION
**Pull request by bogdan-rosianu**
Added `--output-directory` flag in order to specify where the files should be saved

### CODE REVIEW

File: cmd/filegen/main.go [hunk 1]
```diff
@@ -783,3 +797,13 @@ func generateBlockchainHook(converter core.PubkeyConverter) (process.BlockChainH
...
 }
+
+func prepareOutputDirectory(outputDirectory string) error {
+	_, err := os.Stat(outputDirectory)
+
+	if os.IsNotExist(err) {
+		return os.MkdirAll(outputDirectory, 0755)
+	}
+
+	return nil
```
Comment (SebastianMarian) [inline cmd/filegen/main.go:808]: return err ?

Comment (bogdan-rosianu) [reply cmd/filegen/main.go]: done

File: cmd/filegen/main.go [hunk 2]
```diff
@@ -783,3 +797,13 @@ func generateBlockchainHook(converter core.PubkeyConverter) (process.BlockChainH
 
 	return hooks.NewBlockChainHookImpl(arg)
 }
+
+func prepareOutputDirectory(outputDirectory string) error {
+	_, err := os.Stat(outputDirectory)
+
```
Comment (iulianpascalau) [inline cmd/filegen/main.go:803]: remove empty line

Comment (bogdan-rosianu) [reply cmd/filegen/main.go]: done

**Pull request merged**